app/models/utils.rb

```diff
@@ -0,0 +1,35 @@
+module Utils
+  ##
+  # check which parameters and moes have no more than 10 values and return array of ids
+  def self.get_allowed_params(experiment)
```

Comment:
Trzeba wymyślić inną nazwę metody, bo ta nic nie mówi (wiem, że jest komentarz, ale to jest metoda utilsów i nie wiadomo o co chodzi w tym kontekście).